config/settings.js

```diff
@@ -51,11 +67,30 @@ window.spinnakerSettings = {
         region: cfDefaultRegion
       },
     }
+    titan: {
+      defaults: {
+        account: titanPrimaryAccount,
+        region: titanDefaultRegion
+      },
+    },
+    kubernetes: {
+      defaults: {
+        account: kubernetesPrimaryAccount,
+        namespace: kubernetesDefaultNamespace
+      },
+    }
+  },
+  whatsNew: {
```

Comment:
https://github.com/spinnaker/deck/blob/master/settings.js#L50

I honestly just copied it from deck's `settings.js` file to try to sync them up more. Should I remove it?